packages/esm-patient-registration-app/src/patient-registration/field/id/id-field.test.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Identifiers } from './id-field.component';
+import { Resources, ResourcesContext } from '../../../offline.resources';
+import { Form, Formik } from 'formik';
+import { PatientRegistrationContext } from '../../patient-registration-context';
+import { useConfig } from '@openmrs/esm-framework';
+import { mockedIdentifierTypes } from '../__mocks__/identifier-types.mock';
+
+jest.mock('@openmrs/esm-framework', () => ({
+  ...jest.requireActual('@openmrs/esm-framework'),
+  useConfig: jest.fn(),
+}));
+
+describe('Identifiers', () => {
+  const mockResourcesContextValue = {
+    addressTemplate: [],
+    currentSession: {
+      authenticated: true,
+      sessionId: 'JSESSION',
+      currentProvider: { uuid: 'provider-uuid', identifier: 'PRO-123' },
+    },
+    relationshipTypes: [],
+    identifierTypes: [...mockedIdentifierTypes],
+  } as Resources;
+
+  const openmrsID = {
```

Comment:
Can we move this one also to the mocks file?

Reply:
done @Piumal1999